### PR TITLE
Ridge Regression-Base

### PR DESCRIPTION
**Pull request by SSHREYA71**

### CODE REVIEW

File: Ridge Regression/Ridge Regression- Base.py [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+        m = x.shape[0]     #getting the no. of data points
+        
+        # #initialising weights on the basis of number of input parameters
+        
+        self.w = np.zeros((x.shape[1],1))
+        self.b = 0
+        self.x = x
+        self.y = y
+        
+        for _ in range(self.iteration):
```
Comment (ashwani-rathee) [inline Ridge Regression/Ridge Regression- Base.py:32]: I think there would be i here

**Review by ashwani-rathee — COMMENTED** [review]
Everything else looks fine @SSHREYA71

**Comment by SSHREYA71** [comment]
> Everything else looks fine @SSHREYA71

  @ashwani-rathee  Should I create the readme for this , if its good to go?

**Comment by ashwani-rathee** [comment]
@SSHREYA71 Looks good to me @geekquad

**Comment by SSHREYA71** [comment]
> @SSHREYA71 Looks good to me @geekquad

Thank you :). Working on the Readme then.

**Review by geekquad — APPROVED** [review]
Amazing work @SSHREYA71.

**Comment by SSHREYA71** [comment]
> Amazing work @SSHREYA71. 

Thank you so much.  😊

**Pull request merged**